experimental/bionic-stemcell/README.md

```diff
@@ -0,0 +1,12 @@
+# bionic-stemcell
+
+These Ops files replace xenial stemcells with experimental bionic stemcells.
+
+To use them, you'll also need to use the Ops files found in /misc/source-releases since releases are not yet precompiled for bionic:
+
+```
+create-env.sh -o experimental/bionic-stemcell/[IAAS_NAME].yml \
```

Comment:
a general thought: currently we have the IaaS specifics in their own folders. Now we're introducing a new "feature folder" which has IaaS specifics in filenames. Can we make the bionic stemcell adhere to the current IaaS/feature concept, i.e. have the bionic experimental ops-files in the IaaS specific folders?